--- fcgi-2.2.2/configure.in.orig	Tue Nov 27 17:27:13 2001
+++ fcgi-2.2.2/configure.in	Fri May  3 17:18:31 2002
@@ -32,11 +32,6 @@
 AC_SUBST(LIBFCGIXX)
 AC_SUBST(ECHO_CPP)
 
-AC_CHECK_LIB([nsl],       [main]) ac_cv_lib_nsl=ac_cv_lib_nsl_main
-AC_CHECK_LIB([resolv],    [main]) ac_cv_lib_resolv=ac_cv_lib_resolv_main
-AC_CHECK_LIB([socket], 	  [main]) ac_cv_lib_socket=ac_cv_lib_socket_main
-AC_CHECK_LIB([dnet_stub], [main]) ac_cv_lib_dnet_stub=ac_cv_lib_dnet_stub_main
-AC_CHECK_LIB([ieee], 	  [main]) ac_cv_lib_ieee=ac_cv_lib_ieee_main
 
 AC_SUBST(threaded)
 AC_CHECK_LIB([pthread],	  [main], 
